include/deal.II/sundials/ida.h

```diff
@@ -206,8 +206,8 @@ namespace SUNDIALS
    * {
    *   J = A;
    *
-   *   J(0,0) = alpha;
-   *   J(1,1) = alpha;
+   *   J(0,0) += alpha;
+   *   J(1,1) += alpha;
```

Comment:
The Jacobian we are building is of the form `J = alpha I + A`. The previous form was correct, but only because `A` is zero on the diagonal. I think that this form is easier to read. I fixed that in a number of places.